### PR TITLE
Add temporary banner 

### PR DESCRIPTION
**Pull request by AlexJBanks**
Temporary banner promotes welcome letter and discord

reliant on PR #143 first and promotes discord in line with issue #138

### CODE REVIEW

**Comment by AlexJBanks** [comment]
Preview:
image
Further adjustments can be made

File: css/about.scss [hunk 1]
```diff
@@ -74,6 +74,20 @@ h1, h2, h3, h4 {
   }
 }
 
+#freshers-banner {
```
Comment (Danyc0) [inline css/about.scss:77]: Could we make the bar a bit thicker vertically? The text looks a little too close to the edge of the bar to me

**Review by Danyc0 — CHANGES_REQUESTED** [review]
Do we want to apply this to other pages too? Or just About?

**Comment by AlexJBanks** [comment]
> 
> 
> Do we want to apply this to other pages too? Or just About?

I would happily apply to more pages / all pages but i didn't want to go for it if not everyone was on board

File: css/about.scss (hunk 1 above)
Comment (AlexJBanks) [reply css/about.scss:77]: does latest commit resolve?

**Comment by JTWWilson** [comment]
I'd be happy to apply it to all pages (certainly the home page is the big one, I think) as a temporary measure until the start of term.

**Comment by Danyc0** [comment]
Yea I agree, home was the main one I was thinking of too (others/all could also be added too)

File: css/about.scss (hunk 1 above)
Comment (Danyc0) [reply css/about.scss:77]: Yup!

**Comment by AlexJBanks** [comment]
Banner has now been rolled out to all pages 

`index`
image

`\about`
image

**Review by Danyc0 — APPROVED** [review]
Looks good to me

**Pull request merged**